virttest/qemu_devices/qdevices.py

```diff
@@ -1073,8 +1074,20 @@ def hotplug_hmp(self):
         return out
 
     def hotplug_qmp(self):
-        """ :return: the hotplug monitor command """
-        return "object-add", self.params
+        """ :return: the object-add command """
+        params = self.params.copy()
+
+        # qom-type and id are mandatory
+        kwargs = {
+            "qom-type": params.pop("backend"),
+            "id": params.pop("id")
+        }
+
+        # props is optional
+        if len(params) > 0:
+            kwargs["props"] = dict(params)
```

Comment:
I think it's better to  `kwargs["props"] = params `

Reply:
An OrderedDict obj can be JSON serializable.
Change done. 
Thanks.